parts/kubernetesmasteraddons-heapster-service.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
+    task: monitoring
```

Comment:
Just when I took things from the heapster repo they had it. Haven't thought too much further into it.